src/server/eval/unary_operation_node.cpp

```diff
@@ -36,7 +36,7 @@ UnaryOperationNode::UnaryOperationNode(op::UnaryOperationEnum operation, const n
 string UnaryOperationNode::eval(GeoObject* obj) const
 {
 	// This is needed beacase of MSVC
-	assert(false);
```

Comment:
The class isn't even instanciated... it is not mentioned in the eval_parser :) https://github.com/alex85k/alacarte/blob/min-windows-compat/src/server/parser/eval_grammar.cpp